index.html

```diff
@@ -359,19 +410,32 @@ <h2 class="text-black mb-4">Privacy</h2>
         </section>
 
         <!-- FAQ -->
-        <!-- TODO <section class="faq-section bg-light" id="faq">
+        <section class="faq-section bg-light" id="faq">
             <div class="container px-4 px-lg-5">
                 <div class="row gx-4 gx-lg-5 justify-content-center">
                     <div class="col-lg-8">
                         <h2 class="text-black mb-4">FAQ</h2>
-
+                        <h4>My computer is not seen by the headset</h4>
+                        <div>
+                            <p>If the server list is empty in the headset app:</p>
+                            <ul>
+                                <li>Make sure your computer is connected on the same network as your headset.</li>
+                                <li>Check that <mark>avahi</mark> is running with <mark>systemctl status avahi-daemon</mark>. If it is not, enable it with <mark>systemctl enable --now avahi-daemon</mark>.</li>
```

Comment:
Is it possible to have the css for \<mark> to be more visible? Or use plain text for avahi, and something with a copy button for commands to be typed